src/main/java/net/openhft/chronicle/queue/RollCycles.java

```diff
@@ -65,6 +65,10 @@ public enum RollCycles implements RollCycle {
      * 0xffffffff entries per day, indexing every 256th entry, leave as 4K and 256 for historical reasons.
      */
     FAST_DAILY(/*-----------*/"yyyyMMdd'F'", 24 * 60 * 60 * 1000, 4 << 10, 256),
+    /**
+     * 0xffffffff entries per week, indexing every 256th entry, leave as 4K and 256 for historical reasons. Cycle starts Sunday 00:00
+     */
+    WEEKLY(/*-----------*/"YYYY'W'ww", 7 * 24 * 60 * 60 * 1000, 4 << 10, 256, RollCycleArithmetic.SUNDAY_00_00),
```

Comment:
Does it provide much value weekly defaulting to Sunday UTC? that could be as late as 2pm Monday for our friends in Kiribati.

I feel like if you're choosing a weekly roll cycle you really should be nominating an epoch as well.

Reply:
Actually scratch that it would be 2pm Sunday.... this seems reasonable then